Sources/MountebankSwift/Models/Imposter/Imposter.swift

```diff
@@ -48,23 +50,59 @@ public struct Imposter: Codable, Equatable {
         case requests
     }
 
+    /// A server representing a test double.
+    ///
+    /// See: [mbtest.org/docs/api/contracts?type=imposter](https://www.mbtest.org/docs/api/contracts?type=imposter)
+    /// - Parameters:
+    ///   - port: Port to run the imposter on. Defaults to a randomly assigned port that will be returned in the response
+    ///   - networkProtocol: Protocol that the imposter will respond to.
+    ///   - name: Descriptive name that will show up in the logs and the imposters UI.
+    ///   - stubs: A set of behaviors used to generate a response for an imposter. An imposter can have 0 or more stubs,
+    ///     each of which are associated with different predicates and support different responses.
+    ///
+    ///     You would use multiple stubs for an imposter if the types of response you return depends on something in the
+    ///     request, matched with a predicate.
+    ///   - defaultResponse: Allows you to override the default response that Mountebank sends back if no predicate matches
+    ///     a request. Even if a predicate does match but the response isn't fully specified, these values get merged in to complete the
+    ///     response.
```

Comment:
Can't we extend this line length thing for comments started with `///`?

Reply:
Yes that is possible with `ignores_comments`. see https://realm.github.io/SwiftLint/line_length.html

Reply:
Shall we do that? 

And then, do we just let them wrap? I mean, Xcode does a fine job at wrapping right? 

<img width="1061" alt="image" src="https://github.com/MountebankSwift/MountebankSwift/assets/1330668/c6ceb5c3-5820-47f9-ada0-8bf7598769dc">

Or do we still cap them, but when formatted at the 120 mark? 

<img width="1278" alt="Screenshot 2024-01-20 at 23 17 14" src="https://github.com/MountebankSwift/MountebankSwift/assets/1330668/49f7f8f5-0d31-424f-b224-b6ce5f249419">

The only downside of that is that when you make your screen smaller, this happens: 

<img width="1050" alt="image" src="https://github.com/MountebankSwift/MountebankSwift/assets/1330668/339ecc18-ecdc-4c23-a7bf-1d13f94794af">

Reply:
Not sure. Xcode does indeed a good job but I do not know how GitHub/other software will render such long sentences.

Reply:
RIght